pkgs/development/python-modules/soundcard/default.nix

```
@@ -0,0 +1,47 @@
+{
+  buildPythonPackage,
+  cffi,
+  fetchPypi,
+  lib,
+  libpulseaudio,
+  numpy,
+  setuptools,
+  testers,
+}:
+
+buildPythonPackage rec {
+  pname = "SoundCard";
+  version = "0.4.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-QQg1UUuhCAmAPLmIfUJw85K1nq82WRW7lFFq8/ix0Dc=";
+  };
```

IIRC `pname` is no longer allowed to contain uppercase characters. This means you can no longer do `inherit pname` in `fetchPypi`.

```suggestion
  pname = "soundcard";
  version = "0.4.3";
  pyproject = true;

  src = fetchPypi {
    pname = "SoundCard";
    inherit version;
    hash = "sha256-QQg1UUuhCAmAPLmIfUJw85K1nq82WRW7lFFq8/ix0Dc=";
  };
```

---

Though it might also be better to use `fetchFromGitHub` instead of `fetchPypi` since `fetchFromGitHub` is a more easily inspectable source.

---

Thanks for the comment. I was not aware about the `pname` thing. I just pushed a version with all lowercase `pname` and `fetchFromGitHub`.

Anything else I should change before this can be merged?